buildpack-run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Reconstruct repo locally
+git --version
+git init
+git remote add origin https://github.com/jakirkham/miniforge.git
+git fetch origin
+git checkout -fq "$SOURCE_VERSION"
+
+# Construct dummy git user
+git config --global user.name "Your Name"
+git config --global user.email "you@example.com"
```

Comment:
As we are not pushing the changes, these config values for `git` don't matter, but we do need them to make the tag.